framework/encode/vulkan_state_tracker.cpp

```diff
@@ -932,12 +932,12 @@ void VulkanStateTracker::TrackSemaphoreSignalState(uint32_t           wait_count
 void VulkanStateTracker::TrackAcquireImage(
     uint32_t image_index, VkSwapchainKHR swapchain, VkSemaphore semaphore, VkFence fence, uint32_t deviceMask)
 {
-    assert(swapchain != VK_NULL_HANDLE);
```

Comment:
Are there cases now where swapchain might VK_NULL_HANDLE?

Reply:
The swapchain handle is still expected to be a non-null value here.  The assert wasn't completely removed, it was moved below the type cast and combined with a second check.